connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/TunnelChannelListener.java

```diff
@@ -46,14 +57,22 @@ public void channelInitialized(final Channel channel) {
 
     @Override
     public void channelOpenSuccess(final Channel channel) {
+        if (initialWindowSize > 0) {
+            // workaround for Rebex SSH server
```

Comment:
I'm not sure if we should "blame" the specific server in the comment, as it seems so be a problem with the client not waiting for the adjust message and an ending in a deadlock. The behavior sending a zero window size seems to be common across different SSH server implementations.

Reply:
I have adjusted the comment for the workaround.